app/src/main/kotlin/com/wire/android/ui/home/gallery/MediaGalleryScreen.kt

```diff
@@ -70,19 +109,44 @@ fun MediaGalleryContent(imageAsset: ImageAsset.PrivateAsset) {
             .background(colorsScheme().surface)
     ) {
         ZoomableImage(
-            imageAsset = imageAsset,
-            contentDescription = stringResource(R.string.content_description_user_avatar),
+            imageAsset = viewModel.imageAssetId,
+            contentDescription = stringResource(R.string.content_description_image_message),
             imageLoader = imageLoader
         )
     }
 }
 
+@Composable
+private fun getSnackbarMessage(messageCode: MediaGallerySnackbarMessages): Pair<String, String?> {
```

Comment:
Just a question: should we use @Composable for methods where the only reason why this annotation is to use `stringResource` instead of `resources.getString` and providing `resources` as a param? I know it's easier this way and we probably won't use such method outside of composable, but on the other hand, my first impression was: oh, so it's a composable, so why it doesn't display anything but just returns strings 🤔

Reply:
Sounds like a fair request. It stays cleaner this way!